app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -206,7 +207,11 @@ public void onValueChanged(@NonNull Boolean loading) {
                 }
 
                 final MainActivity activity = (MainActivity)getActivity();
-                activity.setCursorEnabled(!getWebView().getUrl().contains("youtube.com/tv"));
+                final IWebView webView = getWebView();
+                // Enable the cursor if website is youtube.com/tv
+                final boolean enableCursor = webView == null ? false :
+                        (webView.getUrl() == null ? false : !webView.getUrl().contains("youtube.com/tv"));
```

Comment:
nit: Add a comment explaining why this is necessary, or that this is a wack-a-mole fix that we don't fully understand.

Reply:
Since we could rewrite this, this is the only really important nit, I think.